### ISSUE TITLE
Fix releasing SwiftUI with Cocoapods

### ISSUE THREAD

**Issue by philipphofmann** [getsentry/sentry-cocoa#2581]
### Description

Releasing SwiftUI with Cocoapods through craft [failed](https://github.com/getsentry/publish/actions/runs/3847774499/jobs/6554662329) with
```
  STDOUT: pod: Updating spec repo `github.com-cocoapods`
  pod: Validating podspec
  pod:  -> SentrySwiftUI
 -> SentrySwiftUI (0.1.0)
  pod:     - ERROR | [SentrySwiftUI/Core] file patterns: The `source_files` pattern did not match any file.
  pod:
  pod: [!] The spec did not pass validation, due to 1 error.
  pod:
```

That's a bit weird as CI validates the podspec with pod lib lint
https://github.com/getsentry/sentry-cocoa/blob/afcb16afb068947ebc45230ff592cbe41ecad3c0/.github/workflows/lint.yml#L71-L83

Also the path seems to be fine
https://github.com/getsentry/sentry-cocoa/blob/afcb16afb068947ebc45230ff592cbe41ecad3c0/SentrySwiftUI.podspec#L26

**Comment by brustolin** [comment]
This is strange. There is nothing wrong with podspec, all tests and lints are passing.

**Comment by brustolin** [comment]
SentryPrivate initial release was very straightforward, so I dont think this is the problem.

**Comment by brustolin** [comment]
solved with #2624
